System/Variables.gd

```diff
@@ -66,3 +66,9 @@ func evidence_keys():
 		if key.ends_with("_name") or key.ends_with("_pic") or key.ends_with("_desc"):
 			ev_keys[key.split("_")[0]] = 1
 	return ev_keys.keys()
+
+func value_replace(value):
```

Comment:
- I think I'd like this alongside keywords - it's more about parsing variables within a script, while System/Variables is more of a low level object about how variables are stored and retrieved
- WSExpression.string_to_bool and WSExpression.bool_to_string probably fit better here though, maybe as static functions. Then get_truth doesn't need to reference anything (ideal world: System/Variables.gd references nothing)